csv-data-management/info.json

```diff
@@ -331,6 +331,124 @@
         }
       ],
       "open": true
+    },
+    {
+      "title": "Join and Extract Data from two CSV",
+      "operation": "join_two_csv_and_extract_data",
+      "description": "This action will extract data by join of two CSV file. User can select option to deduplicate rows based on columns",
+      "parameters": [
+        {
+          "title": "Type",
+          "type": "select",
+          "name": "input",
+          "required": true,
+          "visible": true,
+          "editable": true,
+          "value": "",
+          "options": [
+            "Attachment IRI",
+            "File IRI"
+          ]
+        },
+        {
+          "title": "First File Reference ID",
+          "type": "text",
+          "name": "file_one_value",
+          "required": true,
+          "visible": true,
+          "editable": true,
+          "value": ""
+        },
+        {
+          "title": "First File Column Names",
+          "type": "text",
+          "name": "file1_column_names",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "value": "",
+          "description": "comma separate list of columns name you want to extract from CSV File",
+          "tooltip":"comma separate list of columns name you want to extract from CSV File"
+        },
+        {
+          "title": "Number of rows to skip from First File",
+          "type": "integer",
+          "name": "numberOfRowsToSkipFirst",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "value": null,
+          "description": "Number of rows you want to skip from top of CSV File. Note first row will not be skipped as it has column names",
+          "tooltip":"Number of rows you want to skip from top of CSV File. Note first row will not be skipped as it has column names"
+        },
+        {
+          "title": "Type",
+          "type": "select",
+          "name": "input",
+          "required": true,
+          "visible": true,
+          "editable": true,
+          "value": "",
+          "options": [
+            "Attachment IRI",
+            "File IRI"
+          ]
+        },
+        {
+          "title": "Second File Reference ID",
+          "type": "text",
+          "name": "file_two_value",
+          "required": true,
+          "visible": true,
+          "editable": true,
+          "value": ""
+        },
+        {
+          "title": "Second File Column Names",
```

Comment:
Will have to change other parameters as well such as First File Reference ID to File1: Reference ID. IMO on UI it will show too manny columns . For any user , : looks like split . I will try doing it and see how it looks in UI